ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/NETWORK_PORTS_TESTS.yml

```diff
@@ -0,0 +1,49 @@
+type: l2leaf
+mgmt_gateway: 192.168.0.1
+
+port_profiles:
+  - profile: common
+    mode: access
+    vlans: "999"
+    spanning_tree_portfast: edge
+    spanning_tree_bpdufilter: enabled
+
+  - profile: ap_with_port_channel
+    parent_profile: common
+    vlans: "101"
+    port_channel:
+      mode: active
+
+  - profile: pc
+    parent_profile: common
+    vlans: "100"
+
+network_ports:
+  - switches:
+      - network-ports-tests-1
```

Comment:
I don't see a reason for it not to work given the regex but I would add a test with a hostname with dots so that we can monitor the behavior in the futur